classes/helper/HelperImageUploader.php

```diff
@@ -66,7 +66,7 @@ protected function validate(&$file)
         }
 
         if ($file['size'] > $this->getMaxSize()) {
-            $file['error'] = sprintf(Tools::displayError('File (size : %1s) is too big (max : %2s)'), $file['size'], $this->getMaxSize());
+            $file['error'] = Context::getContext()->getTranslator()->trans('File (size: %1s) is too big (max: %2s)', array($file['size'], $this->getMaxSize()), 'Admin.Notifications.Error');
```

Comment:
There's another occurrence of this sentence which should be updated too. (classes/Uploader.php)